src/cli.rs

```diff
@@ -362,4 +324,32 @@ mod common_args {
         #[clap(long)]
         pub include_time: bool,
     }
+
+    /// Arguments related to defining a block
+    #[derive(Parser)]
+    pub struct BlockArgs {
+        /// The block to append to the token. If `--block` and `--block-file` are omitted, an interactive $EDITOR will be opened.
+        #[clap(long)]
+        pub block: Option<String>,
```

Comment:
Why `clap(long)` while it is `Option<String>` ?

Reply:
This will ask to [clap](https://docs.rs/clap/latest/clap/#) to parse the optional string as a long (all arguments passed to the command line is considered as String).
* No error if no value (because of optional string)
* No error if a valid long is passed 
* Explicit error returned by clap if the value can't be parsed as a valid long.

Reply:
But how the `long` value is stored/accessed then if 'block` is of type `Optional<String>`?
Something like `foo: long = convertToLong(block)` or what?